channels/tsmf/client/tsmf_media.c

```diff
@@ -176,11 +176,12 @@ static TSMF_SAMPLE* tsmf_stream_pop_sample(TSMF_STREAM* stream, int sync)
 	TSMF_STREAM* s;
 	TSMF_SAMPLE* sample;
 	BOOL pending = FALSE;
-	TSMF_PRESENTATION* presentation = stream->presentation;
 
 	if (!stream)
 		return NULL;
 
+	TSMF_PRESENTATION* presentation = stream->presentation;
```

Comment:
That will break older C compilers which require variables to be declared on top of a scope.